.changeset/slow-impalas-tap.md

```diff
@@ -0,0 +1,6 @@
+---
+"blitz": patch
+"@blitzjs/generator": patch
+---
+
+#3748 Add custom template usage back to toolkit
```

Comment:
we should flesh this out more, tell the benefit, and show how to use the new command, etc.